tools/feature_version_audit.py

```
@@ -0,0 +1,685 @@
+        is_core = meta['is_core'] if meta else False
+        if is_core:
+            continue
+        ini_path = get_feature_ini(feature_dir)
+        prior_ver = get_prior_version(ini_path, base_ref) if ini_path else None
+        new_ver = get_version_from_ini(ini_path) if ini_path else None
+        changes = get_changed_files(feature_dir, base_ref)
+        change_types = set(os.path.splitext(f)[1].lower() for _, f in changes)
+        all_commits = []
+        bump_commit = None
+        bump_author = None
+        for status, f in changes:
```

Ruff (0.11.9)

Loop control variable `status` not used within loop body

Rename unused `status` to `_status`

(B007)